src/main/java/com/intuit/graphql/orchestrator/batch/AuthDownstreamQueryModifier.java

```diff
@@ -91,6 +93,12 @@ public TraversalControl visitField(Field node, TraverserContext<Node> context) {
       requireNonNull(fieldDefinition, "Failed to get Field Definition for " + node.getName());
 
       context.setVar(GraphQLType.class, fieldDefinition.getType());
+
```

Comment:
Can we also add integration test i.e. send query to graphql orchestrator and validate the outgoing query.